src/components/MultipleChoice/index.jsx

```diff
@@ -76,12 +77,22 @@ export default class MultipleChoice extends React.Component {
 
   componentWillReceiveProps (nextProps) {
     if (nextProps.error) this.showError();
+    const newChoiceValues = this.getCorrectChoiceValues(nextProps);
+    const oldChoiceValues = this.getCorrectChoiceValues(this.props);
+
     const {query} = this.props;
     const {choices} = this.state;
 
+    if (newChoiceValues !== oldChoiceValues) {
```

Comment:
I believe this will always be true and that the state will always be updated.

Reply:
Yes! I must stringify the objects 👍